02-peer-protocol.md

```diff
@@ -279,8 +288,9 @@ Outputs in the constructed transaction are sorted by `serial_id`.
 `sats` is the satoshi value of the output.
 
 `script` is the scriptPubKey for the output (with its length omitted).
-It's left undefined if you accept other standard outputs such as `OP_RETURN`
-or implement stricter checks on standardness.
+`script`s are not required to follow standardness rules: non-standard
+scripts such as `OP_RETURN` may be accepted, but the corresponding
```

Comment:
Thanks for the change. This now makes sense to me :D
Are OP_RETURNs not standard though?

https://github.com/bitcoin/bitcoin/blob/2ac71d20b2ebbfea76ee7369fddffbd78bd2c010/src/script/standard.cpp#L209

Reply:
Oh right. I was thinking of the data carrier one.